modules/ROOT/pages/configure-adv-tls-context.adoc

```diff
@@ -0,0 +1,95 @@
+= Import a TLS Context from Secrets Manager (Advanced)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If your environment has specific security requirements or requires advanced configuration, you can import a TLS context from the secrets manager.
+
+== Prerequisites
+
+You must have the following permissions to enable inbound traffic to Runtime Fabric:
+
+* Grant access to secrets
+* Manage Secret Groups
+* Read secrets metadata
+* Write secrets
+
+If you do not have access to *Access Management*, ask your organization admin (system administrator) to provide the "Manage Runtime Fabrics" permission.
```

Comment:
It is redundant to ask users to add "Manage Runtime Fabrics". This is a Pre-Req for RTF and not particular to a Secrets Manager TLS Context.

Reply:
Removed this - thank you!